Userland/Libraries/LibUnicode/Normalize.cpp

```diff
@@ -122,7 +122,13 @@ static u32 combine_code_points(u32 a, u32 b)
 {
     Array<u32, 2> const points { a, b };
     // FIXME: Do something better than linear search to find reverse mappings.
-    for (auto const& mapping : Unicode::code_point_decompositions()) {
+    size_t index = { 0 };
+    for (size_t i = 0;; ++i) {
+        auto mapping_maybe = Unicode::code_point_decomposition_by_index(index);
+        ++index;
```

Comment:
:thinking: Instead of two counters, why not just:
```c++
    for (size_t index = 0;; ++index) {
        auto mapping_maybe = Unicode::code_point_decomposition_by_index(index);
```

Reply:
Oops, I thought I had removed that earlier. Should be good now.